docs/tutorials/local_iteration_series/local-Iteration-testing-multiplayer-games-locally.md

```diff
@@ -0,0 +1,62 @@
+---
+id: local_iteration_testing_locally
+title: Local Iteration - Testing multiplayer games locally
+description: Guide covering the available workflows for testing multiplayer games locally.
+---
+
+Testing a multiplayer game presents unique challenges to developers. 
+Currently Unity does not have a first-party streamlined solution to the problem of local iteration while developing a multiplayer game. Developers have to either repeatedly create player builds, or they have to use a symbolic link solution which comes with a set of drawbacks and is not officially supported by Unity.
+
+In lieu of a first-party solution we document the practices that can be used today, along with call-outs for specific risks and things to watch out for.
```

Comment:
"we document the practices that can be used today"
The goal of this PR would be to write the actual doc that'll end up on the website. I wouldn't say "we document the practices", no?